examples/oft/tasks/sendOFT.ts

```diff
@@ -0,0 +1,112 @@
+import { task } from 'hardhat/config'
+import { getEidForNetworkName } from '@layerzerolabs/devtools-evm-hardhat'
+import { addressToBytes32 } from '@layerzerolabs/lz-v2-utilities';
+import { SendParam } from "../utils/typeDefinitions";
+import { Options } from '@layerzerolabs/lz-v2-utilities';
+import { EndpointId } from '@layerzerolabs/lz-definitions';
+
+// send tokens from a contract on one network to another
+task('lz:oft:send', 'Sends tokens from either SwellOFTAdapter or SwellOFT')
```

Comment:
This task seems to be specific to `SwellOFT`. Devtools is a collection of generic, reusable developer utilities.